projects/15_basic_material/main.cpp

```diff
@@ -377,37 +393,92 @@ void ProjApp::SetupMaterials()
     createInfo.bindings.push_back({grfx::DescriptorBinding{CLAMPED_SAMPLER_REGISTER, grfx::DESCRIPTOR_TYPE_SAMPLER, 1, grfx::SHADER_STAGE_ALL_GRAPHICS}});
     PPX_CHECKED_CALL(GetDevice()->CreateDescriptorSetLayout(&createInfo, &mMaterialResourcesLayout));
 
+    // Green metal rust
+    {
+        SetupMaterialResources(
+            "poly_haven/textures/green_metal_rust/diffuse.png",
+            "poly_haven/textures/green_metal_rust/roughness.png",
+            "poly_haven/textures/green_metal_rust/metalness.png",
+            "poly_haven/textures/green_metal_rust/normal.png",
+            mWoodMaterial);
```

Comment:
mWoodMaterial is used below also. Fine since those attributes seem useless, but still.